esp-wifi/src/wifi/mod.rs

```diff
@@ -2356,6 +2484,140 @@ fn apply_sta_config(config: &ClientConfiguration) -> Result<(), WifiError> {
     }
 }
 
+fn apply_sta_eap_config(config: &EapClientConfiguration) -> Result<(), WifiError> {
+    let mut cfg = wifi_config_t {
```

Comment:
Hmm, is there some overlap between normal WPA connections where we can reduce/resuse some code to avoid duplication here?

Reply:
It's just SSID which is common - don't think it's worth it